types/inet_test.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+import (
+	"net"
+	"reflect"
+	"testing"
+
+	_ "github.com/lib/pq"
+)
+
+func TestInetParse(t *testing.T) {
+	inet, err := ParseInet("1.2.3.4/32")
+	if err != nil {
+		t.Error(err)
+	}
+	if !inet.IP.Equal(net.ParseIP("1.2.3.4")) || !reflect.DeepEqual(inet.Mask, net.CIDRMask(32, 32)) {
```

Comment:
This is just a suggestion, not necessary to change:
This checks can be more easily done with something like https://github.com/stretchr/testify. I especially love it because if you have complex structs with a lot of nested messages and the check is not passed, it will print you the diff between compared structs. This is very useful when something is suddenly broken and you try to figure what is wrong.
Just an example from the [toolkit](https://github.com/infobloxopen/atlas-app-toolkit/blob/master/gorm/filtering_test.go).